src/index.ts

```diff
@@ -1,12 +1,73 @@
-import { Application } from 'probot';
+import assert from 'assert';
+import express from 'express';
```

Comment:
Do we really need to import `express` and require it as a peer dependency in `package.json` just to reference the `Request` and `Response` types?